docs/releasing/publishing-a-pre-release.md

```diff
@@ -46,20 +46,78 @@ Projects can install `govuk-frontend@beta` in their package.json alongside other
 
    - Use `prerelease` to bump from `v5.0.0-beta.0` to `v5.0.0-beta.1`
 
-7. Apply the new pre-release version number by running:
+6. Create and check out a new branch (`release-[version-number]`). See the [versioning documentation](/docs/contributing/versioning.md) for more information.
+
+7. If you're publishing a beta pre-release, update the [`CHANGELOG.md`](/CHANGELOG.md) by:
+
+   - changing the 'Unreleased' heading to the new version number and release type. For example, '5.0.0-beta.0 (Pre-release)'
+   - adding a new 'Unreleased' heading above the new version number and release type, so users will know where to add PRs to the changelog
+   - saving your changes
+
+8. Apply the new pre-release version number by running:
 
    ```shell
-   npm version <PRE-RELEASE TYPE> --preid <PRE-RELEASE TAG> --no-git-tag-version --workspace govuk-frontend
+   npm version <PRE-RELEASE TYPE> --preid <PRE-RELEASE IDENTIFIER> --no-git-tag-version --workspace govuk-frontend
    ```
 
    This step will update the [`package.json`](/package.json) and project [`package-lock.json`](/package-lock.json) files.
 
    Do not commit the changes.
 
-8. Run `npm run pre-release` to create and push a new branch that contains your changes. This process may take a few moments and will display a `Success!` message.
+9. Update browser data from ["Can I use"](https://caniuse.com) by running:
+
+   ```shell
+   npx update-browserslist-db@latest
+   ```
+
+   This step will update the project [`package-lock.json`](/package-lock.json) file if updates are found.
+
+10. Run `npm run build-release` to:
+
+    - build GOV.UK Frontend into the [`/package`](/package) and [`/dist`](/dist) directories
+    - commit the changes
+    - push a branch to GitHub
+
+    You will now be prompted to continue or cancel.
+
+11. If you're publishing a beta pre-release, create a pull request
```

Comment:
**note** Figured we don't really need to reverse `internal` pre-releases to `main`, only `beta` ones. Reason for that is that if we alternate `beta` and `internal` pre-releases (for example, to check a bugfix before the next pre-release), that will make the version bump rocky for `beta` release as we'd go from `-beta.X` to `-internal.0` then the next `prerelease` bump would go to `-beta.0` instead of `-beta.X+1`.